docs/UserGuide.md

```diff
@@ -275,34 +279,38 @@ Examples:
 </div>
 
 ### Editing an applicant : `edit`
+[Table of contents](#table-of-contents)
 
 Edits an applicant's with specified index in RecruitIn.
 
 Format: `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL_ADDRESS] [r/ROLE] [et/EMPLOYMENT_TYPE]
  [s/EXPECTED_SALARY] [l/LEVEL_OF_EDUCATION] [y/YEARS_OF_EXPERIENCE] [t/TAG] [i/INTERVIEW] [nt/NOTES]`
  
- * Edit command must take at least 1 prefix input.
+ * Edit command must take at least 1 prefix parameter.
  * The `INDEX` refers to the index number shown in the displayed applicants list.
  * For `t/` prefix in particular, if **only** a single tag prefix is provided like so `t/` with no values, it will erase
 remove tags from the applicant.
 <div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
-Giving more than 1 tag prefix input with 1 or more having no value will instead lead to an error. (i.e. `edit 1 t/ t/smart` leads to an error)
+Giving more than 1 tag prefix parameter with 1 or more having no value will instead lead to an error. (i.e. `edit 1 t/ t/smart` leads to an error)
 </div>
- * Inputs for each prefix is taken as a single value. (i.e. `r/software engineer` has the value `software engineer`)
+ * Parameters for each prefix is taken as a single value. (i.e. `r/software engineer` has the value `software engineer`)
 
 Examples:
-* `edit 1 r/Software Engineer` will change the ***role*** of the applicant with the index number 1
-* `edit 1 t/` will remove all ***tags***s from the applicant with index number 1
+* `edit 1 t/` will remove all ***tag***s from the applicant with index number 1
 * `edit 1 n/John t/` will change the name of the applicant with index number 1 to `John` and remove all the applicant's ***tag***s
+* `edit 1 r/Software Engineer` will change the ***role*** of the applicant with the index number 1
+Before: ![images](images/EditBefore.png)
```

Comment:
The "Before: " is presented next to the sentence in line 301 in the UG, I think you need to add an extra line between line 301 and 302.